bosh_cli/lib/cli/commands/ssh.rb

```diff
@@ -6,7 +6,7 @@ module Command
     class Ssh < Base
 
       # bosh ssh
-      usage 'ssh'
+      usage 'ssh <job>/<index>'
```

Comment:
I'm not actually even sure that the above code would work for the /-separated job/index form, since that's only three arguments and the function *requires* four.  And making the `destination` flag optional leads to bad documentation (since it's not really optional)